package.json

```diff
@@ -20,33 +20,35 @@
     "lint:prettier": "prettier --check 'packages/**/*.js'",
     "prettier:format": "prettier --write 'packages/**/*.js'"
   },
-  "prettier": "@thorgate/prettier-config",
   "devDependencies": {
-    "@babel/core": "^7.12.1",
-    "@typescript-eslint/eslint-plugin": "^4.9.0",
-    "@typescript-eslint/parser": "^4.9.0",
-    "babel-eslint": "^10.1.0",
-    "babel-preset-razzle": "^4.0.4",
-    "eslint": "^6.8.0",
-    "eslint-config-airbnb": "^18.2.1",
-    "eslint-config-airbnb-typescript": "^12.0.0",
-    "eslint-config-prettier": "^8.0.0",
-    "eslint-config-react-app": "^5.2.0",
-    "eslint-import-resolver-typescript": "^2.3.0",
-    "eslint-plugin-flowtype": "^4.6.0",
-    "eslint-plugin-import": "^2.20.1",
-    "eslint-plugin-jest": "^24.1.3",
-    "eslint-plugin-jsx-a11y": "^6.2.3",
-    "eslint-plugin-prettier": "^3.2.0",
-    "eslint-plugin-react": "^7.19.0",
-    "eslint-plugin-react-hooks": "^4.2.0",
-    "jest": "^26.6.3",
-    "lerna": "^3.22.1",
+    "@babel/core": "^7.16.0",
```

Comment:
Also need to update readme files which list the peer deps install commands with package versions.